Update mypy to 0.930 and fix newly discovered type errors
